src/main/java/org/springframework/data/neo4j/core/mapping/Constants.java

```diff
@@ -28,7 +28,7 @@
  * @soundtrack Milky Chance - Sadnecessary
  * @since 6.0
  */
-@API(status = Status.INTERNAL, since = "6.0")
+@API(status = Status.EXPERIMENTAL, since = "6.0")
```

Comment:
Any reasons for this change?

Reply:
So that people can use that in their queries to refer to the root object.